ibis/expr/operations/temporal.py

```diff
@@ -258,9 +258,17 @@ class TimestampFromYMDHMS(Value):
     hours: Value[dt.Integer]
     minutes: Value[dt.Integer]
     seconds: Value[dt.Integer]
+    nullable: bool = True
```

Comment:
This is leaking datatype information into the operation. We made a decision a long time ago to store nullability on datatypes.

What is the rationale here?